RHEL/7/input/oval/package_autofs_removed.xml

```diff
@@ -5,10 +5,10 @@
     <metadata>
       <title>Package autofs Removed</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_rhel</platform>
```

Comment:
And another OVAL placed into the very same directory is instructed to be multi-platform. Can you explain why?